test/teos/e2e/test_basic_e2e.py

```diff
@@ -525,3 +534,79 @@ def test_appointment_shutdown_teos_trigger_while_offline():
     assert appointment_info.get("status") == "dispute_responded"
 
     teosd_process.terminate()
+
+
+# def test_get_all_appointments_watcher(api, client, get_all_db_manager):
```

Comment:
Why are this two commented out?

Reply:
get_all_appointments was moved from the client to the cli, so I thought I'd just figure out how to do proper cli and rpc tests in the next PR together with all the other cli commands.
It is used anyway in the e2e test, so it is not completely not covered.